src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -655,25 +654,27 @@ else if (option == ImportOrderOption.INFLOW) {
      * @param isStatic whether the token is static or not.
      * @param previous previous non-static but current is static (above), or
      *                  previous static but current is non-static (under).
-     * @param line the line of the current import.
+     * @param ast the line of the current import.
```

Comment:
If this stays, this isn't "the line" anymore.

Reply:
done.